packages/@datapull/aws-origin/index.js

```diff
@@ -48,25 +48,55 @@ class AwsOrigin {
     }
     console.log('[AWS Origin] params', params);
 
-    return new Promise((resolve, reject) => {
-      serviceClient[this.config.method].call(serviceClient, params, async (err, resp) => {
-        if (err) {
-          console.error('[AWS Origin] ERROR ', err);
-          return reject(err);
+    const mergeResults = (newResponse, prevResponse) => {
+      Object.keys(newResponse).forEach(key => {
+        if (Array.isArray(prevResponse[key]) && Array.isArray(newResponse[key])) {
```

Comment:
AWS API doesn't have a unified `results` field in list responses. Instead it would output results in fields like `Accounts` or `Instances`. Here we just merge the results of two list responses together without knowing which field has the results in advance.

Reply:
Gotcha.